android/build.gradle

```diff
@@ -2,15 +2,16 @@ plugins {
     id 'com.android.library'
     id 'org.jetbrains.kotlin.android'
     id 'org.jetbrains.kotlin.plugin.serialization'
+    id("org.jetbrains.kotlinx.kover")
```

Comment:
```suggestion
    id('org.jetbrains.kotlinx.kover')
```

Reply:
btw, why add `kover` ?

Reply:
It's not required since we do not currently measure coverage for android as a part of checks, but added for consistency since in newer version of kover [it should be applied](https://kotlin.github.io/kotlinx-kover/gradle-plugin/) to each submodule for which we might want a coverage report.

Reply:
But we didn't have this for lib as well, right? Maybe I misunderstand something, but I don't think we should add this. We are running codecov on gh actions, I think this should be enough

Reply:
I applied it for lib as well, the gh actions fail otherwise. Newer versions of kover won't generate a report to upload to codecov if kover is not explicitly applied to submodule.